static/src/js/util/subscriptions.js

```diff
@@ -16,5 +16,5 @@ export const prepareSubscriptionQuery = (params) => {
   delete prunedParams.sortKey
 
   // Return the remaining parameters stringified, don't encode the values because CMR won't accept them
-  return stringify(prunedParams, { encode: false })
+  return stringify(prunedParams, { encode: false, indices: false, arrayFormat: 'brackets' })
```

Comment:
There are some granule search parameters in CMR that are indexed. e.g. https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html#g-passes.

Reply:
Will this code handle the indexed case?

Reply:
We don't support those parameters, do you know of any others?

Reply:
Good. That is only one on granules. But if this function would also be used for collection searches, you are in trouble as science-keywords parameter are indexed.

Reply:
Cool, this code is only for subscriptions, so only granule queries

Reply:
Based on where this code is located, I don't think it is used for collection searches.